### PR TITLE
Validate Password function is added

### PR DESCRIPTION
**Pull request by aarushisoni**

### CODE REVIEW

File: moto/cognitoidp/models.py [hunk 1]
```diff
@@ -39,6 +42,10 @@ class UserStatus(str, enum.Enum):
     UNCONFIRMED = "UNCONFIRMED"
     RESET_REQUIRED = "RESET_REQUIRED"
 
+class InvalidPasswordException(Exception):
```
Comment (bblommers) [inline moto/cognitoidp/models.py:45]: Please add this Exception to `exceptions.py`, having all exceptions in one place makes it much more readable. 

Also please check it follows the same standard as the other exceptions in that file: extend JSONRestError, passing a message that explains what's going wrong (`This password was invalid`), etc.

Comment (aarushisoni) [reply moto/cognitoidp/models.py:46]: Update exceptions.py #5758 
kindly review this.

**Comment by bblommers** [comment]
It looks like the build is failing - see https://github.com/spulec/moto/actions/runs/3672930949/jobs/6212952807

You can run the following commands locally, to see if it would pass:


```
flake8 moto/cognitoidp
black --check moto/cognitoidp
pylint moto/cognitoidp
mypy moto/cognitoidp
pytest tests/test_cognitoidp
```

You could also run these commands for the entire codebase, but that would take a bit more time:

```
make test
```

**Review by bblommers — APPROVED** [review]
Awesome - thanks @aarushisoni!

**Pull request merged**